### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by AlexHamn**
Added some basic styling.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Comic+Neue:wght@400;700&display=swap" rel="stylesheet"/>
     <title>Leaderboard</title>
 </head>
```
Comment (rloterh) [inline src/index.html:10]: - [x] _Stored values_ are **duplicating** in the output when a couple of entries are being made. Kindly note that these values were entered just once but some of them are displaying 4 times in the results. Kindly ensure to improve upon this aspect. 

**Duplicated values/ results:**
image

Comment (rloterh) [inline src/index.html:10]: - [x] Your and _ReadMe_ is **not descriptive** and **incomplete**. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this project,  which will likely feature on your portfolio. Kindly ensure to improve upon your ReadMe. You may use [this template](https://github.com/microverseinc/readme-template) to craft a professional ReadMe but remember to customize it to your project.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Alex, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- Nice UI design
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @AlexHamn, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**